funnel/views/notifications/comment_notification.py

```diff
@@ -19,9 +19,61 @@
     NewCommentNotification,
     User,
 )
-from ...transports.sms import OneLineTemplate
+from ...transports.sms import OneLineTemplate, SmsTemplate
```

Comment:
Where is OneLineTemplate being used?

Reply:
It is being used in the CommentReportNotification